homeassistant/components/flux_led/light.py

```diff
@@ -365,12 +381,25 @@ def white_value(self):
         """Return the white value of this light."""
         return self._white_value
 
+    @property
+    def temperature_cw(self):
+        """Return the cold white temperature."""
+        rgbww = self._bulb.getRgbww()
```

Comment:
Does this do I/O?

Reply:
Yes but we wrapped _bulb in the async during setup entry.

Reply:
Please make it a method instead of a property. I/O calls should be explicit. Accessing a property is not expected to do I/O.

Reply:
OK - updated and pulled the IO into a method which is called by the two properties that need it.